pkg/v1/tkg/tkgpackageclient/package_install.go

```diff
@@ -32,6 +32,28 @@ const (
 
 // InstallPackage installs the PackageInstall and its associated resources in the cluster
 func (p *pkgClient) InstallPackage(o *tkgpackagedatamodel.PackageOptions, progress *tkgpackagedatamodel.PackageProgress, operationType tkgpackagedatamodel.OperationType) {
+	p.installPackage(o, progress, operationType)
+}
+
+// InstallPackageSync installs the PackageInstall and its associated resources in the cluster and returns an error if any
```

Comment:
Would it be possible to remove equivalent block from the cmd layer and call ```*sync``` version from there directly to avoid code duplication?

Otherwise, could you please provide a TODO comment on cmd layer for ```package/package_install.go``` that they can be refactored to use ```InstallPackageSync``` to avoid code duplication? (cause now certain blocks of code would be duplicated in both places).

Same comment applies to other operations with ```*sync``` equivalents.

Reply:
Makes sense. I have updated the `cmd` package to use `sync` equivalents.